package.json

```diff
@@ -17,10 +19,14 @@
 	"devDependencies": {
 		"@types/node": "^20.14.2",
 		"dotenv-cli": "^7.4.2",
+		"eslint": "^9.6.0",
+		"eslint-config-prettier": "^9.1.0",
+		"eslint-plugin-prettier": "^5.1.3",
+		"eslint-plugin-simple-import-sort": "^12.1.1",
 		"prettier": "^3.3.2",
-		"prettier-plugin-organize-imports": "^3.2.4",
 		"ts-node": "^10.9.2",
-		"typescript": "^5.4.5"
+		"typescript": "^5.4.5",
```

Comment:
Is this a good chance to update to TS 5.5+? Or would you like to do that on a new PR?

Reply:
I'd prefer to update `typescript` and other dependencies on a separate PR.